apps/collabora.sh

```diff
@@ -23,10 +19,78 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+# Check if collabora is already installed
+print_text_in_color "$ICyan" "Checking if Collabora is already installed..."
+if does_this_docker_exist 'collabora/code'
+then
+    choice=$(whiptail --radiolist "It seems like 'Collabora' is already installed.\nChoose what you want to do.\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+    "Uninstall Collabora" "" ON \
+    "Reinstall Collabora" "" OFF 3>&1 1>&2 2>&3)
+
+    case "$choice" in
+        "Uninstall Collabora")
+            print_text_in_color "$ICyan" "Uninstalling Collabora..."
+            # Check if Collabora is previously installed
+            # If yes, then stop and prune the docker container
+            docker_prune_this 'collabora/code'
+
+            # Disable RichDocuments (Collabora App) if activated
+            if [ -d "$NC_APPS_PATH"/richdocuments ]
+            then
+                occ_command app:remove richdocuments
+            fi
+            
+            msg_box "Collabora was successfully uninstalled."
+            exit
+        ;;
+        "Reinstall Collabora")
+            print_text_in_color "$ICyan" "Reinstalling Collabora..."
+            
+            # Check if Collabora is previously installed
+            # If yes, then stop and prune the docker container
+            docker_prune_this 'collabora/code'
+            
+            # Disable RichDocuments (Collabora App) if activated
+            if [ -d "$NC_APPS_PATH"/richdocuments ]
+            then
+                occ_command app:remove richdocuments
+            fi
+        ;;
+        *)
+        ;;
+    esac
+else
+    print_text_in_color "$ICyan" "Installing Collabora..."
+fi
+
+# Check if OnlyOffice is previously installed
+# If yes, then stop and prune the docker container
+docker_prune_this 'onlyoffice/documentserver'
+
+# remove OnlyOffice-documentserver if activated
+install_if_not jq
+if occ_command_no_check app:list --output=json | jq -e '.enabled | .documentserver_community' > /dev/null
+then
+    msg_box "Removing OnlyOffice-documentserver: $1\nYou will be given the option to abort when you hit OK."
```

Comment:
To noisy if you ask me. Just let the user know that Onlyoffice will be uninstalled with a countdown, and the option to abort with CTRL+C. It's always possible to reinstall it later if that's the intention. 

This just takes time, and is un-smooth.

Reply:
Okay, you are right.

Reply:
this should be done.